plugin/evm/message/handshake/upgrade_config_test.go

```diff
@@ -0,0 +1,35 @@
+package handshake
+
+import (
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/params"
+	"github.com/ava-labs/subnet-evm/precompile/contracts/nativeminter"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSerialize(t *testing.T) {
```

Comment:
can we add more test cases with: 
* Enabled/Admin addresses
* Initial configs (like mint config)
* State Upgrades

Reply:
I think we need to make sure:
* all of our precompiles can be deserializable
* add serialize tags to precompile config template
* if possible add a test to `config_test.go`so that new precompiles can be tested out when generated.

Reply:
^^^

Reply:
This is still pending.

Reply:
I'm writing more tests now